src/uct/tcp/tcp_ep.c

```diff
@@ -313,22 +296,31 @@ ucs_status_t uct_tcp_ep_create(const uct_ep_params_t *params,
     return status;
 }
 
-void uct_tcp_ep_mod_events(uct_tcp_ep_t *ep, uint32_t add, uint32_t remove)
+void uct_tcp_ep_mod_events(uct_tcp_ep_t *ep, int add, int remove)
 {
-    int old_events = ep->events;
-    int new_events = (ep->events | add) & ~remove;
+    uct_tcp_iface_t *iface = ucs_derived_of(ep->super.super.iface,
+                                            uct_tcp_iface_t);
+    int old_events         = ep->events;
+    int new_events         = (ep->events | add) & ~remove;
+    ucs_status_t status;
 
     if (new_events != ep->events) {
         ep->events = new_events;
         ucs_trace("tcp_ep %p: set events to %c%c", ep,
-                  (new_events & EPOLLIN)  ? 'i' : '-',
-                  (new_events & EPOLLOUT) ? 'o' : '-');
+                  (new_events & UCS_EVENT_SET_EVREAD)  ? 'i' : '-',
```

Comment:
maybe convert the letters in the trace log from 'i', 'o' to 'r', 'w'?

Reply:
Changed `i`,`o` to `r`,`w`.